src/server/server.cc

```diff
@@ -1252,17 +1252,8 @@ std::string Server::GetKeyspaceInfo(const std::string &ns) {
   KeyNumStats stats;
   GetLatestKeyNumStats(ns, &stats);
 
-  // FIXME(mwish): output still requires std::tm.
-  auto last_scan_time = static_cast<time_t>(GetLastScanTime(ns));
-  std::tm last_scan_tm{};
-  localtime_r(&last_scan_time, &last_scan_tm);
-
-  string_stream << "# Keyspace\r\n";
```

Comment:
I think it should not be removed.

Reply:
```
string_stream << "# Keyspace\r\n";
```

Reply:
Yes, I wrongly removed this line. Thank you!